dist/kothic-leaflet.js

```diff
@@ -135,5 +141,20 @@ L.TileLayer.Kothic = L.TileLayer.Canvas.extend({
         script.charset = 'utf-8';
         document.getElementsByTagName('head')[0].appendChild(script);
         return script;
+    },
+
+    _loadJSON: function(url, zoom, x, y) {
+        var xhr = new XMLHttpRequest();
+        xhr.onreadystatechange = function() {
+            if (xhr.readyState == XMLHttpRequest.DONE) {
+                if (xhr.status == 200) {
+                    window.onKothicDataResponse(JSON.parse(xhr.responseText), zoom, x, y);
```

Comment:
What will we do about this? Should I just merge this and we fix everything else later?